src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,135 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.Objects;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
     @Override
     public void add(T value) {
+        Node<T> newNode = new Node<>(value);
```

Comment:
```suggestion
        Node<T> newNode = new Node<>(tail, value, null);
```
![image](https://github.com/mate-academy/jv-linked-list/assets/20326858/3426095d-c92f-4c6e-a3cd-7ad4b10cc255)

Reply:
I'm sorry, I forgot about checklist under the task